parser.js

```diff
@@ -61,7 +62,7 @@ var fields = {
 var keys = Object.keys(fields);
 
 function _parseLine (str) {
-    var reg = /^(\w+\s+\d+\s+\S+)\s+(\S+)\s+(\S+)\[(\d+)\]:\s+(\S+):(\d+)\s+\[(\S+)\]\s+(\S+)\s+(\S+)\/(\S+)\s+(\S+)\/(\S+)\/(\S+)\/(\S+)\/(\S+)\s+(\S+)\s+(\S+) *(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\/(\S+)\/(\S+)\/(\S+)\/(\S+)\s+(\S+)\/(\S+)\s+"(\S+)\s+([^"]+)\s+(\S+)" *$/gi
+  var reg = /^(\w+\s+\d+\s+\S+)\s+(\S+)\s+(\S+)\[(\d+)\]:\s+(\S+):(\d+)\s+\[(\S+)\]\s+(\S+)\s+(\S+)\/(\S+)\s+(\S+)\/(\S+)\/(\S+)\/(\S+)\/(\S+)\s+(\S+)\s+(\S+) *(\S+)\s+(\S+)\s+(\S+)\s+(\S+)\/(\S+)\/(\S+)\/(\S+)\/(\S+)\s+(\S+)\/(\S+)\s+({.*}\s+)?"(\S+)\s+([^"]+)\s+(\S+)" *$/gi
```

Comment:
I suggest doing `({.*})?\s*` instead of `({.*}\s+)?`, then you'll not have spaces in the end of captured group.